visma/functions/variable.py

```diff
@@ -77,3 +77,177 @@ def integrate(self, wrtVar):
 
     def calculate(self, val):
         return self.coefficient * ((val**(self.power)))
+
+    def __radd__(self, other):
+        return self + other
+
+    def __add__(self, other):
+        from visma.functions.constant import Constant
+        if isinstance(other, Variable):
+            sortedValuesSelf = self.value.sort()
```

Comment:
Powers will have to be reordered based on the `value` sort.

Reply:
intended to use `sorted()` function instead of `sort()` did that by mistake. Now used sorted so the order of `value` list is never changed. Rectified.

Reply:
The powers should also be reordered, given variable y^3z^2x will have value property as `[y, z, x]` and power as `[3, 2, 1]`.
After value sort the powers should be reordered from `[3, 2, 1]` to `[1, 3, 2]`.